core/variant/variant_utility.cpp

```diff
@@ -471,6 +475,10 @@ double VariantUtilityFunctions::move_toward(double from, double to, double delta
 	return Math::move_toward(from, to, delta);
 }
 
+double VariantUtilityFunctions::rotate_toward(double from, double to, double delta) {
```

Comment:
Minor note: These should be `p_`, but everything else in the file is also missing them. So keep your PR as-is, I'm just making a note so that this can be cleaned up in a future PR.